client/landing/stepper/declarative-flow/internals/steps-repository/woo-confirm/index.tsx

```diff
@@ -168,13 +167,7 @@ const WooConfirm: Step = function WooCommerceConfirm( { navigation } ) {
 			`/checkout/${ wpcomDomain }/${ upgradingPlan?.product_slug ?? '' }`
 		),
 		productName,
-		description: productName
-			? sprintf(
-					/* translators: %s: The upgrading plan name (ex.: WordPress.com Business) */
-					__( 'Upgrade to the %s plan and set up your WooCommerce store.' ),
-					productName
-			  )
-			: __( 'Upgrade to set up your WooCommerce store.' ),
+		description: __( 'Upgrade to the Pro plan and set up your WooCommerce store.' ),
```

Comment:
If we're going to use a static string, we could also get rid of all the references to `productName`.

Reply:
I've removed the property from the object, but I left the variable since it looks related to this issue https://github.com/Automattic/wp-calypso/issues/63296